backpack/core/derivatives/conv_transposend.py

```diff
@@ -52,47 +52,53 @@ def _bias_jac_mat_prod(self, module, g_inp, g_out, mat):
         return jac_mat.expand(*expand_shape)
 
     def _weight_jac_mat_prod(self, module, g_inp, g_out, mat):
-        V = mat.shape[0]
-        G = module.groups
-        C_in = module.input0.shape[1]
-        N = module.output.shape[0]
-        C_out = module.output.shape[1]
-
-        mat_reshape = mat.reshape(V, C_in, G, C_out // G, *module.weight.shape[2:])
-        u = unfold_by_conv_transpose(module.input0, module).reshape(
-            N, C_in // G, G, *module.weight.shape[2:], *module.output.shape[2:]
-        )
-
-        dims_kern = "xyz"[: self.conv_dims]
-        dims_data = "abc"[: self.conv_dims]
-        einstr = "nig{0}{1},vigo{0}->vngo{1}".format(dims_kern, dims_data)
-        jac_mat = einsum(einstr, u, mat_reshape)
-
-        return self.reshape_like_output(jac_mat, module)
+        if module.groups == 1:
+            V = mat.shape[0]
+            G = module.groups
+            C_in = module.input0.shape[1]
+            N = module.output.shape[0]
+            C_out = module.output.shape[1]
+
+            mat_reshape = mat.reshape(V, C_in, G, C_out // G, *module.weight.shape[2:])
+            u = unfold_by_conv_transpose(module.input0, module).reshape(
+                N, C_in // G, G, *module.weight.shape[2:], *module.output.shape[2:]
+            )
+
+            dims_kern = "xyz"[: self.conv_dims]
+            dims_data = "abc"[: self.conv_dims]
+            einstr = "nig{0}{1},vigo{0}->vngo{1}".format(dims_kern, dims_data)
+            jac_mat = einsum(einstr, u, mat_reshape)
+
+            return self.reshape_like_output(jac_mat, module)
+        else:
+            raise NotImplementedError("Groups greater than 1 are not supported yet")
```

Comment:
I would revert the order: raise the exception if `groups==1`. Then you avoid indenting the actual code.